MAINTAINERS.md

```diff
@@ -10,7 +10,6 @@ describes governance guidelines and maintainer responsibilities.
 | Nick Young | [youngnick](https://github.com/youngnick) | [VMware](https://www.github.com/vmware/) |
 | Steve Sloka | [stevesloka](https://github.com/stevesloka) | [VMware](https://www.github.com/vmware/) |
 | Steve Kriss | [skriss](https://github.com/skriss) | [VMware](https://www.github.com/vmware/) |
-| Alex Xu |[xaleeks](https://github.com/xaleeks)| [VMware](https://www.github.com/vmware/) |
```

Comment:
Removal of a maintainer needs to go through the regular process outlined in the governance doc, so please revert this change.

Reply:
yup already discussed that, will fix